internal/pkg/cli/deploy/workload.go

```diff
@@ -738,6 +738,18 @@ func envFiles(unmarshaledManifest interface{}) map[string]string {
 	return nil
 }
 
+// ContainerDependencies returns a map of ContainerDependency objects from workload manifest.
+func ContainerDependencies(unmarshaledManifest interface{}) map[string]manifest.ContainerDependency {
```

Comment:
We might want to move this to the `manifest` pkg. Any reason we want to keep it here?